.github/workflows/rust.yml

```
@@ -2,132 +2,96 @@ name: Rust CI
+        cargo test --workspace --all-features --no-run
+        cargo nextest run --all-features
+
+    - name: Generate coverage (llvm-cov)
+      uses: taiki-e/install-action@v2
+      with:
+        tool: cargo-llvm-cov
+    - run: |
+        cargo llvm-cov --workspace --all-features --output-path ./coverage --lcov --output-format=lcov
+
+    - name: Upload to Codecov
+      uses: codecov/codecov-action@v3
```

actionlint (1.7.4)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)